store/store.go

```diff
@@ -59,12 +59,12 @@ func RegisterStore(s Store) error {
 func GetStore() (Store, error) {
 	name := config.Name
 	if name == "" {
-		return nil, errors.New("store Name is empty")
+		return nil, errors.New("store name is empty")
 	}
 
 	store, ok := StoreSlots[name]
 	if !ok {
-		return nil, errors.New("no such name Store")
+		return nil, errors.New("store not found")
```

Comment:
it's better to give the name to the not found store

Reply:
done